travis_after_all.py

```diff
@@ -2,6 +2,7 @@
 import json
 import time
 import logging
+from functools import reduce
```

Comment:
sorry, have lack of knowledge.
is this something specific for python 3.x? reduce import is not required for 2.7